### DISCUSSION TITLE
Alfen Power Total not showing on energy dashboard

### DISCUSSION THREAD

**Discussion by ojoojoo** [egnerfl/alfen_wallbox#25] [Q&A]
Hi,

I cannot add this entity to the energy dashboard under Individual devices:
sensor.alfen_proline_duo_achter_active_power_total

According to this article https://www.home-assistant.io/docs/energy/faq/#troubleshooting-missing-entities the device class should be energy instead of power.

state_class: measurement
unit_of_measurement: W
device_class: power
icon: mdi:circle-slice-3
friendly_name: Alfen Proline Duo Achter Active Power Total

Basically what I want is to display the amount of kWh what is charged per day per socket to the energy dashboard.
Is there a way to get this done?
